packages/next/src/server/render.tsx

```diff
@@ -1416,6 +1416,17 @@ export async function renderToHTMLImpl(
     }
   }
 
+  const dynamicCSSFiles = new Set<string>()
+  Object.values(reactLoadableManifest).forEach(({ files }) => {
+    if (files) {
+      files.forEach((file) => {
+        if (file.endsWith('.css')) {
+          dynamicCSSFiles.add(file)
+        }
+      })
+    }
+  })
```

Comment:
This includes all dynamically loaded CSS of the whole application. Could become quite large...

Reply:
As we handle it server-side through context, I think the amount of all dynamic CSS would be all right as it'll be smaller than `react-loadable-manifest`.

I'm worried that this computation will be held N time of server render, which is redundant.

How about generating another manifest, `dynamic-css-manifest`, while generating the `react-loadable-manifest` and using it for server-side context?